monitor/api/pom.xml

```diff
@@ -20,13 +20,320 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>eu.seaclouds-project</groupId>
-    <artifactId>monitor</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
-  </parent>
-  <groupId>eu.seaclouds-project</groupId>
-  <artifactId>monitor-api</artifactId>
-
-</project>
\ No newline at end of file
+    <parent>
+        <artifactId>monitor</artifactId>
+        <groupId>eu.seaclouds-project.monitor</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>monitor-api</artifactId>
+
+	<developers>
+		<developer>
+			<name>Dionysis Athanasopoulos</name>
+		</developer>
+	</developers>
+
+	<repositories>
+        <repository>
+            <id>maven2-repository.java.net</id>
+            <name>Java.net Repository for Maven</name>
+            <url>http://download.java.net/maven/2/</url>
+            <layout>default</layout>
+        </repository>
+
+		<repository>
+			<id>deib-polimi-releases</id>
+			<url>https://github.com/deib-polimi/deib-polimi-mvn-repo/raw/master/releases</url>
+		</repository>
+
+		<repository>
+			<id>deib-polimi-snapshots</id>
+			<url>https://github.com/deib-polimi/deib-polimi-mvn-repo/raw/master/snapshots</url>
+		</repository>
+
+		<repository>
+        	<id>imperial-modaclouds-releases</id>
+        	<url>https://github.com/imperial-modaclouds/modaclouds-data-collectors/releases/download/1.3-Snapshot</url>
```

Comment:
The link is broken (maybe https://github.com/imperial-modaclouds/modaclouds-data-collectors/releases/download/1.3-Snapshot/data-collector-1.3-SNAPSHOT.jar ?). However, I don't think single URL's to files can work as maven repositories.